### ISSUE TITLE
Menu theme Override affects admin menu

### ISSUE THREAD

**Issue by wannaco** [impresspages/ImpressPages#735]
![image](https://cloud.githubusercontent.com/assets/7078331/8396994/8a7e6794-1d79-11e5-8e45-b044d06d3405.png)

the class is inserted as if it were part of the webiste's menu generation.

https://www.impresspages.org/docs/view
Even removes attributes as specified in the override view, this a big impact .

![image](https://cloud.githubusercontent.com/assets/7078331/8397016/4361ce72-1d7a-11e5-8c7a-8b9d7eaa50e7.png)

**Comment by maskas** [comment]
Yes. That's an important discovery. Admin should use its own view files.

**Comment by bekoli** [comment]
+1 good catch
